src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -44,15 +48,21 @@ const ResearchProfile = () => {
   const dispatch = useDispatch();
   const { t } = useTranslation();
   const history = useHistory();
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+  const [registrationsPage, setRegistrationsPage] = useState(0);
+  const [projectsPage, setProjectsPage] = useState(0);
+  const [projectRowsPerPage, setProjectRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+  const [registrationRowsPerPage, setRegistrationRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
 
   const currentCristinId = useSelector((store: RootState) => store.user?.cristinId) ?? '';
   const isPublicPage = history.location.pathname === UrlPathTemplate.ResearchProfile;
   const personId = isPublicPage
     ? new URLSearchParams(history.location.search).get('id') ?? '' // Page for Research Profile of anyone
     : currentCristinId; // Page for My Research Profile
 
+  const personIdNumber = isPublicPage
+    ? new URLSearchParams(history.location.search).get('id')?.split('/').pop() ?? ''
+    : getIdentifierFromId(currentCristinId ?? '');
```

Comment:
Skal ikke denne alltid samsvare med verdien over?
```suggestion
  const personIdNumber = getIdentifierFromId(personId);
```